packages/medusa/src/api-v2/admin/products/validators.ts

```diff
@@ -679,3 +678,44 @@ export class ProductTypeReq {
   @IsString()
   value: string
 }
+
+export class ProductVariantPricesCreateReq {
+  @IsString()
+  currency_code: string
+
+  @IsInt()
+  amount: number
+
+  @IsOptional()
+  @IsInt()
+  min_quantity?: number
+
+  @IsOptional()
+  @IsInt()
+  max_quantity?: number
```

Comment:
this is missing `rules` for a price. This can be region_id or sales_channel_id or pretty much anything else.

Reply:
@riqwan Should a price set have rules though? Or should we reserve that only for price lists? I don't want this to be full-blown pricelist model IMO. WDYT?

Reply:
True, not the price set, but the prices under the price set.

Ex.

EUR price is 400 where rules = {} // default price for EUR
FR price is 300 where rules = { region_id: "france" }

Reply:
@riqwan sorry I wasn't clear, I meant should the money amounts under a price set have rules, or should we reserve that only for price lists' money amounts, and have the price set money amounts just be defaults per currency?

Reply:
It would deviate from how we use price lists currently.

For the bulk variants editor in product admin you can set default currency price and region prices for a variant. How would that look like in this API with the delineation?

Reply:
@riqwan I haven't seen the new designs so I am not sure, but I guess we would need to support it. Let me add a TODO and I'll tackle it separately, maybe we would change this to have a different shape than it has now, adding a rule option.